packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -12354,6 +12417,8 @@ module ZIndexes {
 // WARNING: Unsupported export: MAX_COLOR_VALUE
 // WARNING: Unsupported export: MAX_COLOR_RGBA
 // WARNING: Unsupported export: ColorPicker
+// WARNING: Unsupported export: ColorRectangle
+// WARNING: Unsupported export: ColorSlider

```

Comment:
Component conformance test is failing on this 2 exports. I would try having `ColorRectangle` and `ColorSlider` exports being done from the `index` of the ColorPicker folder and no need to have a separate index file for each subComponent. This should fix your test failure.

Reply:
I was missing the top-level component definition files. I've added these and the build works locally, just waiting for Travis now. 😄

Reply:
Wait... do we want to expose these subcomponents to the top-level? Can they be used separately?

Reply:
I'm not sure how to get the build to work otherwise. At first I just added the exports to ColorPicker/index.ts but then the build failed saying that top-level imports were missing for those components.

Reply:
Good point. I think the issue I had was because I was exporting the component and not just the types, as Shimmer does. I've tried exporting just the types (following the Shimmer pattern) and it builds locally, so hopefully Travis passes. Thanks!